tests/robot-tests/tests/admin_and_public/bau/publish_content.robot

```diff
@@ -59,6 +59,8 @@ Check glossary info icon appears on release preview
     user waits until page contains button    Absence
 
 Click glossary info icon and validate glossary entry
+    user closes user testing modal
```

Comment:
I assume this modal is temporary. Is there a ticket for removing the user testing modal? If so, I'd leave a comment here and on the ticket that this line needs to be removed from the tests.

Reply:
It's temporary but I don't think there is a ticket yet